GeoCAT-comp-examples/matplotlib/climatology_average_example.py

```diff
@@ -0,0 +1,117 @@
+"""
+climatology_average_example.py
+==============================
+This script illustrates the following concepts:
+    - Usage of geocat-comp's `climatology_average <https://geocat-comp.readthedocs.io/en/latest/user_api/generated/geocat.comp.climatologies.climatology_average.html#geocat.comp.climatologies.climatology_average>`_ function
+    - Usage of geocat-datafiles for accessing NetCDF files
+    - Creating a figure with stacked subplots
+
+See following GitHub repositories to see further information about the function and how to access
+data:
+    - For GeoCAT functions: https://github.com/NCAR/geocat-comp
+    - For atm.20C.hourly6-1990-1995.nc file: https://github.com/NCAR/geocat-datafiles/tree/main/netcdf_files
+
+Dependencies:
+    - cftime
+    - geocat.comp
+    - geocat.datafiles (Not necessary for figure but for accessing the data file)
```

Comment:
Would it be more accurate to say "necessary for figure but not for function" because you need the data file in order to create the figure below, but you don't necessarily need the file to use the function? Or possibly just changing the comment to be "for accessing datafile"?

Reply:
I'm trying to express that a user does not need `geocat.datafiles` if they have a different way to access their data. I'll replace it with "for accessing datafile only"